src/components/Home.js

```diff
@@ -26,7 +28,9 @@ const Home = ({ navigation }) => {
 			<View style={{ flex: 1, justifyContent: 'center', alignItems: 'center', flexDirection: 'column', gap: 2 }}>
 				<Coins containerStyle={styles.scoreContainer}/>
 				<StoreButton navigation={navigation} />
+				<SettingsButton navigation={navigation} />
 				<Play navigation={navigation} />
+				<LearnStack navigation={navigation} />
```

Comment:
LearnStack doesn't correspond to what's inside the component, perhaps naming it something like LearnButton could be better?